src/policies/policies.py

```diff
@@ -30,22 +50,44 @@ def validate_resource():
 
     resource_id = resource.attributes[0]['Value']
     user_name = subject.attributes[0]['Value']
-    
-    #To be expanded when implementing more complex policies
+
+    if "Issuer" in subject.attributes[0].keys():
+        issuer = subject.attributes[0]['Issuer']
+    else:
+        issuer = None
+
+    handler_status_issuer = None
+
+    if issuer is not None:
+        handler_status_issuer, handler_issuer_message = ScimHandler.get_instance().modifyAuthServerUrl(issuer)
+
     #For now it serves only as a check if the user attributes were reachable on the AS
     #handler_user_attributes uses this schema: https://gluu.org/docs/gluu-server/4.1/api-guide/scim-api/#/definitions/User
 
     # Call to be used later in development
     try:
-        handler_status, handler_user_attributes = ScimHandler.get_instance().getUserAttributes(user_name)
-    except Exception as e:
-        print("Error While retrieving the user attributes")
+        if handler_status_issuer == 200 or (issuer is None):
+            handler_status, handler_user_attributes = ScimHandler.get_instance().getUserAttributes(user_name)
 
-    if not isinstance(handler_user_attributes, dict):
-        handler_user_attributes = {}
+            if not isinstance(handler_user_attributes, dict):
+                handler_user_attributes = {}
+                for i in range(0, len(subject.attributes)):
+                    handler_user_attributes[subject.attributes[i]['AttributeId']] = subject.attributes[i]['Value']
 
-        for i in range(0, len(subject.attributes)):
-            handler_user_attributes[subject.attributes[i]['AttributeId']] = subject.attributes[i]['Value']
+            if issuer is None:
+                use_env_var = validate_auth_server_url()
+                if use_env_var is True:
+                    handler_user_attributes['issuer'] = str(os.environ["PDP_AUTH_SERVER_URL"])
```

Comment:
This loading of configuration should be aligned to the changes on the PEP too. The expected behavior should be to load config file by default, then override values with those environment variables that are not empty (and in this case then override issuer with whatever is in the XACML request!)